generators/server/files.js

```diff
@@ -343,7 +343,7 @@ const serverFiles = {
         {
             path: DOCKER_DIR,
             condition: generator => generator.buildTool === 'maven',
```

Comment:
I wonder why we added this condition, we should add them for gradle too, isn't us?

Reply:
This is my fault... 
The first release of this blueprint was supporting Maven only so I've added this condition.
Make not sense anymore

Reply:
It's made sense for sure. It's just that I don't want to ask people to change something I'm not sure. 

If you are ok @joewhite101 we could use this PR to remove this condition. I think there is nothing special to do, unless remove the condition and test.